packages/embark/src/cmd/cmd.js

```diff
@@ -174,6 +175,28 @@ class Cmd {
       });
   }
 
+  exec() {
+    program
+      .command('exec [environment] [script|directory]')
+      .description(__("Executes specified scripts or all scripts in 'directory'"))
+      .action((env, target) => {
+        embark.exec({
+          env,
+          target
+        }, (err) => {
+          if (err) {
+            console.error(err.message ? err.message : err);
+            process.exit(1);
+          }
+          console.log('Done.');
+          // TODO(pascal): Ideally this shouldn't be needed.
+          // Seems like there's a pending child process at this point that needs
+          // to be stopped.
+          process.exit(0);
```

Comment:
As mentioned in the comment: once the blockchain module is loaded the process will run forever because we don't tell it to stop. Needs investigation